Resources/Prototypes/_NF/Reagents/chemicals.yml

```
@@ -4,3 +4,33 @@
   desc: reagent-desc-salicylicacid
   physicalDesc: reagent-physical-desc-powdery
   color: "#EEEEEE"
+
+- type: reagent
+  id: JuiceThatMakesYouOOOK
```

```suggestion
  id: JuiceThatMakesYouOok
```
For ease of finding this stuff, avoid allcaps outside of maybe acronyms.